src/nitpick/style.py

```diff
@@ -133,6 +135,12 @@ def parse_cache_option(cache_option: str) -> tuple[CachingEnum, timedelta | int]
     return caching, expires_after
 
 
+def raise_gitlab_incorrect_url_error(url: furl) -> NoReturn:
+    """Raise an error if the URL is not a valid GitLab URL."""
+    message = f"Invalid GitLab URL: {url}"
+    raise ValueError(message)
```

Comment:
This could be a private method on the `GitLabURL` below since it's only used there.